test/templates/bespoke.ts

```diff
@@ -972,6 +972,32 @@ describe("Bespoke template's browser context", () => {
             }))
         })
       })
+
+      describe('Drag resize', () => {
+        it('resizes on drag', () =>
+          testPresenterView(({ deck }) => {
+            Object.defineProperty(document.documentElement, 'clientWidth', {
+              value: 1000,
+            })
+            $p(classes.dragbar).dispatchEvent(new MouseEvent('mousedown'))
+            $p(classes.container).dispatchEvent(
+              new MouseEvent('mousemove', { clientX: 200 })
+            )
+            expect(
+              $p(classes.container).style.getPropertyValue(
+                '--bespoke-marp-presenter-split-ratio'
+              )
+            ).toBe('20%')
+            $p(classes.container).dispatchEvent(new MouseEvent('mouseup'))
+          }))
+        it('no resize without drag', () =>
+          testPresenterView(({ deck }) => {
+            $p(classes.container).dispatchEvent(
+              new MouseEvent('mousemove', { clientX: 200 })
+            )
+            expect($p(classes.container).style.gridTemplateColumns).toBe('')
```

Comment:
No meaning test now. Refer `--bespoke-marp-presenter-split-ratio` instead.

Reply:
fixed in e03b5aab